app/main.py

```diff
@@ -96,19 +91,19 @@ def example_compose(template_id: str, compose_file_schema: ComposeSchema = Query
 def _compose(db: Session, jinja_env: JinjaEnv, template_static_directory: str,
              compose_retrieval_function: Callable[[Template], dict], template_id: str, file_name: str,
              compose_schema: ComposeBaseSchema, accept_header: str | None) -> StreamingResponse:
-    accept_header = accept_header or PDF_MIME
+    accept_header = accept_header or MIMETypeEnum.PDF_MIME.value
     mime_type = get_best_match(accept_header, ALL_AVAILABLE_MIME_TYPES)
 
     if mime_type is None:
         raise UnsupportedMIMEType(accept_header)
 
-    if mime_type == PNG_MIME:
+    if mime_type == MIMETypeEnum.PNG_MIME.value:
         raise PNGCompositionUnavailable()
 
-    if (compose_schema.width is not None or compose_schema.height is not None) and mime_type != PNG_MIME:
+    if (compose_schema.width is not None or compose_schema.height is not None) and mime_type != MIMETypeEnum.PNG_MIME.value:
         raise UnsupportedResizingException(mime_type)
 
-    if compose_schema.page is not None and mime_type != PNG_MIME:
+    if compose_schema.page is not None and mime_type != MIMETypeEnum.PNG_MIME.value:
```

Comment:
i think you don't need to assert with the .value:

<img width="1149" height="532" alt="image" src="https://github.com/user-attachments/assets/e80f2c3c-dc38-4cd0-a3a7-dd4b9291f314" />